evaluation/webarena/run_infer.py

```
@@ -32,7 +32,14 @@
 def get_sandbox():
     global docker_ssh_box
     if docker_ssh_box is None:
-        docker_ssh_box = DockerSSHBox()
```

This should throw an error. right?